frontend/mobile/lib/pages/landing.dart

```diff
@@ -20,32 +21,49 @@ class LandingPage extends StatefulWidget {
 class _LandingPageState extends State<LandingPage> {
   bool isLoggedIn = false;
   int _selectedIndex = 2;
+  String language = "en";
```

Comment:
use the locale variable below instead, all the storage function are already named "locale" and imo it's a better name